prow/images/test-runner/Dockerfile

```diff
@@ -53,6 +53,9 @@ RUN curl -LO https://github.com/tektoncd/cli/releases/download/v${TKN_VERSION}/t
 RUN gem install mixlib-config -v 2.2.4  # required because ruby is 2.1
 RUN gem install mdl -v 0.5.0
 
+# Extra tools through pip
+RUN pip install --user yamllint && mv /root/.local/bin/yamllint /usr/local/bin/
```

Comment:
isnt that pip default  to /usr/local, not sure why we have to do the mv but it doesnt really matter :)

Reply:
@chmouel for some reason, it is creating the file in `/root/.local/bin` :sweat: